### PR TITLE
docs: add a link for the Druid SQL tutorial 

### PR DESCRIPTION
**Pull request by ektravel**
## Description

This PR add a link for the Druid SQL tutorial that uses Jupyter to the `tutorial-jupyter-index.md` file.

This PR can be merged after PR [13465](https://github.com/apache/druid/pull/13465), PR [13342](https://github.com/apache/druid/pull/13342), and PR [13588](https://github.com/apache/druid/pull/13588).

This PR has:

- [ x  ] been self-reviewed.

### CODE REVIEW

**Review by 317brian — APPROVED** [review]
This LGTM. 

For reference,  the diff between this and https://github.com/apache/druid/pull/13342 is that 13342 only includes the link to the API tutorial Jupyter notebook. This one includes the API tutorial and the SQL ( line 65 in `README.md` and line 68 in `tutorial-jupyter-index.md`. This was done in a separate PR rather than a commit to 13342 to make managing dependencies between PRs easier.

File: docs/tutorials/tutorial-jupyter-index.md [hunk 1]
```diff
@@ -68,4 +69,7 @@ The notebooks are located in the [apache/druid repo](https://github.com/apache/d
 
 The links that follow are the raw GitHub URLs, so you can use them to download the notebook directly, such as with `wget`, or manually through your web browser. Note that if you save the file from your web browser, make sure to remove the `.txt` extension.
 
-- [Introduction to the Druid API](https://raw.githubusercontent.com/apache/druid/master/examples/quickstart/jupyter-notebooks/api-tutorial.ipynb) walks you through some of the basics related to the Druid API and several endpoints.
\ No newline at end of file
+- [Introduction to the Druid API](https://raw.githubusercontent.com/apache/druid/master/examples/quickstart/jupyter-notebooks/api-tutorial.ipynb) walks you through some of the basics related to the Druid API and several endpoints.
+- [Introduction to Druid SQL](https://raw.githubusercontent.com/apache/druid/master/examples/quickstart/jupyter-notebooks/sql-tutorial.ipynb) covers the basics of Druid SQL.
+<<<<<<< HEAD
```
Comment (ektravel) [inline docs/tutorials/tutorial-jupyter-index.md:74]:
```suggestion
```

**Comment by techdocsmith** [comment]
@ektravel , do you want to remove the lines referring to Segment sizing and partitioning so we can merge this PR? Thanks!

**Comment by ektravel** [comment]
@techdocsmith I've removed the lines referring to Segment sizing and partitioning. Thank you for fixing the merge conflict.

**Review by techdocsmith — APPROVED** [review]
LGTM

**Pull request merged**